Editorial (#228)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

to contain the superset of all claims the user of the
signed JWT might want to disclose to Verifiers at some point. The
ability to selectively disclose a subset of these claims depending on
the Verifier becomes crucial to ensure minimum disclosure and prevent
Verifiers from obtaining claims irrelevant for the transaction at hand.
SD-JWTs defined in this document enable such selective disclosure of JWT claims.

One example of a multi-use JWT is a verifiable credential, an issuer-signed
credential that contains the claims about a subject, and whose authenticity can be
cryptographically verified.

Similar to the JWT specification on which it builds, this document is a product of the
Web Authorization Protocol (oauth) working group. However, while both JWT and SD-JWT
have potential OAuth 2.0 applications, their utility and application is certainly not constrained to OAuth 2.0.
JWT was developed as a general-purpose token format and has seen widespread usage in a
variety of applications. SD-JWT is a selective disclosure mechanism for JWT and is
similarly intended to be general-purpose specification.

While JWTs for claims describing natural persons are a common use case,

> the mechanisms defined in this document can be used for many other use

Reviewer: ```suggestion
the mechanisms defined in this document can be used for other use
```

## Feature Summary

* This specification defines

>   - a format for an Issuer-signed JWT that enables issuing selectively disclosable claims,

Reviewer 2: ```suggestion
  - a format for an Issuer-signed JWT containing selectively disclosable claims,
```

## Mandatory digest computation of the revealed claim values by the Verifier

ToDo: add text explaining mechanisms that should be adopted to ensure that

> Verifiers validate the claim values received in HS-Disclosures JWT by calculating the

Reviewer: ```suggestion
Verifiers validate the claim values received in Disclosures by calculating the
```

> digests of those values and comparing them with the digests in the SD-JWT:

Reviewer 2: This change breaks the sentence above. I suggest to **remove this section completely:** It comes from a time when the format did force Verifiers to hash the disclosures. Also, manipulation of disclosures is discussed below.

Author: agreed to remove.
